cli/check.translation.php

```diff
@@ -23,17 +23,17 @@
 $displayReport = array_key_exists('r', $options);
 
 $isValidated = true;
-$result = array();
-$report = array();
+$result = [];
+$report = [];
 
+/** @var string $language */
```

Comment:
Please try to make sure that `$languages` has a known type before arriving there, instead of manually giving the type.
I think just making sure that `$options['l']` is a string should be enough